user/deployment/snaps.md

```diff
@@ -1,38 +1,54 @@
+[Snapcraft](https://snapcraft.io/) lets you distribute to all Ubuntu releases and a [growing set of Linux distributions](https://docs.snapcraft.io/core/install) with a single artefact. You publish and update at your pace while still reaching everyone; you're not locked to the release cycle of Ubuntu or any other distribution. The updates apply automatically and roll back if anything goes wrong. They're secure; each update is cryptographically signed and is tamper-proof once installed. The applications are locked down using the same container primitives found in Docker and LXD.
 
-To automate continuous delivery of snaps to the Ubuntu store:
+To upload your snap, add the following to your `.travis.yml`:
 
-1. [Encrypt](https://docs.travis-ci.com/user/encrypting-files) your snapcraft credentials with Travis CI
-2. Add the encrypted credentials to the repository where you have the `snapcraft.yaml` metadata for your package.
-3.  Run `snapcraft` and `snapcraft push` in your `.travis.yml`:
+```yaml
+deploy:
+  provider: snap
+  snap: my.snap
+  channel: edge
+  skip_cleanup: true
+```
+{: data-file=".travis.yml"}
 
-    ```yaml
-    deploy:
-      skip_cleanup: true
-      provider: script
-      script: docker run -v $(pwd):$(pwd) -w $(pwd) snapcore/snapcraft sh -c "apt update && snapcraft && snapcraft push *.snap --release edge"
-      on:
-        branch: master
-    ```
+In order for Travis CI to have permission to upload the snap on your behalf, you need to export a Snap Store login token. You can do this with the snapraft command-line tool, once you have [enabled snap support](https://docs.snapcraft.io/core/install) on your system.
 
-The `snapcraft enable-ci travis` command will assist you getting the credential, encrypting it and adding the right script to the deploy section of your `.travis.yml` file. Here is a [tutorial that will guide you setting up the continuous delivery from Travis CI](https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci#0) on your project.
+```bash
+sudo snap install snapcraft --classic
+```
 
-Note that the `edge` channel is intended for crowdtesting with your community of early adopters. With this deployment script in Travis, every time a pull request lands into the master branch, a new snap will be published to edge. Your testers can install it in any of the supported Linux distros with:
+Login tokens can attenuate how, when, and where they can be used, thus minimising damage from compromise. For Travis CI, export a token that can only upload this snap to the channel you specified above (in this example, `edge`):
```

Comment:
Might be worth using something other than attenuate, for clarity, what do you think?

Reply:
Agreed, the word "specify" probably makes more sense, here. Fixed.